### DISCUSSION TITLE
Using a spritesheet object instead of json

### DISCUSSION THREAD

**Discussion by ScreenCactus** [pixijs/pixijs#7959] [Q&A] [answered]
Hello PixiJS community. 

I am trying to load a spritesheet where the data is already defined like in the below example. How could I load this?

When I run the below code I keep getting (setting '_texture').  Perhaps I am doing this wrong. I am new to PixiJS!

```javascript
let spritesheet = {meta: {...}, frames: {...};
let texture = Texture.from(spriteSheet.meta.image); 
let sheet = new Spritesheet(texture, spritesheet);
```

**Comment by bigtimebuddy** [comment, marked as answer]
You're almost there! Here's a simple example:
https://jsfiddle.net/bigtimebuddy/d1cq79wj/

FYI, make sure you call `parse` on the sheet, this is what actually creates all the textures.

Hope that helps clarify.

**Reply by miltoncandelero** [reply]
🤦‍♀️ I totally forgot that you need to call parse
Sorry @ScreenCactus

**Reply by ScreenCactus** [reply]
Thanks everyone! The example was very useful.

**Comment by domis86** [comment]
related PR with documentation change: https://github.com/pixijs/pixijs/pull/8321
